backend/src/appointment/database/models.py

```diff
@@ -80,9 +81,15 @@ class InviteStatus(enum.Enum):
     revoked = 2  # The code is no longer valid and cannot be used for sign up anymore
 
 
+def encrypted_type(column_type, length: int = 255, **kwargs) -> StringEncryptedType:
```

Comment:
I snuck this in to reduce visual noise. We can eventually add some functionality to not encrypt local dev environments for better debugging.